siatest/testnode.go

```diff
@@ -58,14 +58,14 @@ func NewCleanNode(nodeParams node.NodeParams) (*TestNode, error) {
 	tn := &TestNode{*s, *c, ""}
 
 	// Init wallet
-	wip, err := tn.WalletInitPost("", false)
+	wip, err := tn.WalletInitPost("", "english", false)
```

Comment:
I'm not sure if it's worth supporting alternate dictionaries. AFAIK no one uses anything but english. We can preserve compatibility in the actual API server, but we don't necessarily need to encourage it anywhere else.